backend/src/users/users.resolver.ts

```diff
@@ -8,34 +8,34 @@ import { UsersService } from './users.service';
 
 @Resolver(() => User)
 export class UserResolver {
-  constructor(private readonly usersService: UsersService) {}
-
-  @Query(() => User, { name: 'user', nullable: true })
-  async getUser(@Args() getUserArgs: GetUserArgs): Promise<User> {
-    return this.usersService.user(getUserArgs);
-  }
-
-  @Mutation(() => User)
-  async createUser(
-    @Args('createUserData') createUserData: CreateUserInput,
-  ): Promise<User> {
-    return this.usersService.createUser(createUserData);
-  }
-
-  @Mutation(() => User)
-  async updateUser(
-    @Args('updateUserData') updateUserData: UpdateUserInput,
-  ): Promise<User> {
-    return this.usersService.updateUser({
-      where: updateUserData,
-      data: updateUserData,
-    });
-  }
-
-  @Mutation(() => User)
-  async deleteUser(
-    @Args('deleteUserData') deleteUserData: DeleteUserInput,
-  ): Promise<User> {
-    return this.usersService.deleteUser(deleteUserData);
-  }
+  // constructor(private readonly usersService: UsersService) {}
```

Comment:
Why are these commented out? If this is being deprecated then it should be removed otherwise it should be changed to work with the new changes.

Reply:
I believe these were some boilerplate code that nestjs generated with the setup of the backend. I don't think these would be of any use for now so I will delete them.